wp-content/themes/missilethreat/assets/_scss/abstracts/_placeholders.scss

```diff
@@ -4,3 +4,149 @@
 /*==============================
 =          Placeholders        =
 ==============================*/
+%font-ui-text-sm {
+  font-size: 12px;
```

Comment:
For styles in this file that just use a single size, can we use the `font-size` mixin to convert them to rems?